test/CI-blackbox/renderer.js

```diff
@@ -0,0 +1,78 @@
+
+const {JavaGenerator, JavaScriptGenerator, TypeScriptGenerator, FormatHelpers} = require('../../lib');
+const fs = require('fs').promises;
+const path = require('path');
+
+/**
+ * Processes the input and renders the language the models
+ */
+async function processInput() {
+  const languagesToInclude = process.argv.slice(2);
+  if(languagesToInclude.length === 0) {
+    console.error(`No languages provided, please provide one or more of the following languages: js, ts, java.`);
+    process.exit(0);
+  }
+  const inputFileContent = await fs.readFile(path.resolve(__dirname, './dummy.json'));
+  const input = JSON.parse(inputFileContent);
+  for (const language of languagesToInclude) {
+    if (language === 'java') {
+      //Because of the nature of Java we have to write each model to separate files
+      const generator = new JavaGenerator();
+      const outputPath = './java';
+      const generatedModels = await generator.generate(input);
+      await renderModelsToSeparateFiles(generatedModels, outputPath);
+    } else {
+      let generator;
+      let outputPath;
+      switch(language){
+        case 'js':
+          generator = new JavaScriptGenerator();
+          outputPath = './js/output.js';
+        break;
+        case 'ts':
+          generator = new TypeScriptGenerator();
+          outputPath = './ts/output.ts';
+        break;
+        default:
+          throw new Error(`Does not understand language input ${language}`);
+      }
+      const generatedModels = await generator.generate(input);
+      await renderModels(generatedModels, outputPath);
```

Comment:
You can put it in one switch :)

Reply:
True, It just looks weird when JS and TS works differently from Java 😄